### PR TITLE
New error codes

### PR DESCRIPTION
**Pull request by NoComment1105**
See https://github.com/discord/discord-api-docs/pull/5200 merged in https://github.com/discord/discord-api-docs/commit/722b86a67aac4ba663c43da6f2fd46d5ef446c21 and https://github.com/discord/discord-api-docs/pull/5128 merged in https://github.com/discord/discord-api-docs/commit/e950176b8df67b8d49653ee63e307c0d8a337546

### CODE REVIEW

File: rest/src/main/kotlin/json/JsonErrorCode.kt [hunk 1]
```diff
@@ -229,6 +229,9 @@ public enum class JsonErrorCode(public val code: Int) {
     /** Max number of thread participants has been reached (1000). */
     MaxThreadParticipants(30033),
 
+    /** Max number of daily application command creates has veeb reached (200). */
```
Comment (Lukellmann) [inline rest/src/main/kotlin/json/JsonErrorCode.kt:232]:
```suggestion
    /** Max number of daily application command creates has been reached (200). */
```

File: rest/src/main/kotlin/json/JsonErrorCode.kt [hunk 2]
```diff
@@ -274,6 +277,9 @@ public enum class JsonErrorCode(public val code: Int) {
     /** The user is banned from this guild. */
     UserBannedFromGuild(40007),
 
+    /** Connection has been revoked. */
+    ConnectionRevoked(40008),
```
Comment (Lukellmann) [inline rest/src/main/kotlin/json/JsonErrorCode.kt:281]:
```suggestion
    ConnectionRevoked(40012),
```

File: rest/src/main/kotlin/json/JsonErrorCode.kt [hunk 3]
```diff
@@ -439,6 +445,9 @@ public enum class JsonErrorCode(public val code: Int) {
 
     /** Ownership cannot be transferred to a bot user. */
     OwnershipCannotBeTransferredToBot(50132),
+
+    /** Failed to resize the asset below the maximum size: 262144. */
+    FailedToResizeAssetBelowMaximumSize(50133),
```
Comment (Lukellmann) [inline rest/src/main/kotlin/json/JsonErrorCode.kt:450]:
```suggestion
    FailedToResizeAssetBelowMaximumSize(50138),
```

File: rest/src/main/kotlin/json/JsonErrorCode.kt [hunk 4]
```diff
@@ -503,6 +512,12 @@ public enum class JsonErrorCode(public val code: Int) {
     /** Failed to create stage needed for stage event. */
     FailedToCreateStage(180002),
 
+    /** Message was blocked by automatic moderation. */
+    MessageBlockedByAutomaticModeration(180003),
```
Comment (Lukellmann) [inline rest/src/main/kotlin/json/JsonErrorCode.kt:516]:
```suggestion
    MessageBlockedByAutomaticModeration(200000),
```

File: rest/src/main/kotlin/json/JsonErrorCode.kt [hunk 5]
```diff
@@ -503,6 +512,12 @@ public enum class JsonErrorCode(public val code: Int) {
...
+
+    /** Title was blocked by automatic moderation. */
+    TitleBlockedByAutomaticModeration(180004),
```
Comment (Lukellmann) [inline rest/src/main/kotlin/json/JsonErrorCode.kt:519]:
```suggestion
    TitleBlockedByAutomaticModeration(200001),
```

**Review by Lukellmann — CHANGES_REQUESTED** [review]
I think you got some codes wrong :)

While you are at it, could you also include https://github.com/discord/discord-api-docs/commit/28f0fd6f6d743af360f9fe677c918635c5c7f087 and https://github.com/discord/discord-api-docs/commit/72fb7a35250d952a5a6109a2bd3b78fd928d8418?

**Comment by NoComment1105** [comment]
That's copilot screwing me over lmao. I knew I should've ignored that

**Review by Lukellmann — APPROVED** [review]

**Pull request merged**